agent/lib/src/agent.dart

```diff
@@ -157,6 +157,11 @@ class Agent {
     });
   }
 
+  Future<void> saveDetailFile(String taskKey, String detailFilename) async {
+    await uploadLogChunk(taskKey, '\n\nUploading task details from $detailFilename');
+    await cpFileToGcs(detailFilename, 'gs://flutter-dashboard-task-detail/$taskKey/');
```

Comment:
@keyonghan do you remember the problem with gsutil concurrency when called from dart? I remember it was blocking the agents and had to revert.

Reply:
I don't remember we ever merged PRs with gsutil. Last time I quitted the gsutil PR is due to not working from Windows bot, and stackdriver was chosen as an alternative for viewing task log.

Reply:
Ah, so the "else" clause in `cpFileToGcs()` doesn't capture Windows?

What's the plan to move forward for Windows support? This PR would break the Windows agents if your claim is true.